### ISSUE TITLE
Edit Folder UI displays default "Full Rescan Interval (s)" value instead of the actual one

### ISSUE THREAD

**Issue by scienmind** [syncthing/syncthing#4944]
Scenario:
 - Set "Full Rescan Interval (s)" to something other than default 3600s (watcher enabled), for example to 9000s (2 h 30 min)
 - Save the change and confirm it is written properly in config.xml
 - Restart syncthing
 - Open UI, review folder info
 - Folder info shows rescan at 2h 30min correctly
 - Open Edit Folder dialog
 - Switch to "Advanced" tab
 * Actual result: "Full Rescan Interval (s)" shows 3600
 * Expected result: "Full Rescan Interval (s)" shows 9000 as saved in config.xml

Clicking "Save" at this point will save 3600 to the config, even though the user has never entered such value.

Reproduction isn't 100%, and my impression is that the more folders devices configured, and the slower everything is, the more likely the issue is to occur.
The occurrence conditions are somewhat similar to the issue 4915 (stignore is accidentaly overwritten by web UI on slow machines). Although the two probably aren't closely related.

### Version Information

Syncthing Version: v0.14.47
OS Version: Windows 10 x64
Browser Version: SyncTrayzor's UI

**Comment by uok** [comment]
Yes, there seems to be a small bug. I see "1h" in folder view, which is correct. In "advanced" tab as described above I see "60". If you repeat these steps it shows "3600" correctly. Also it happens only once after page is (re)loaded.

**Comment by imsodin** [comment]
Yup, the mechanism to set `rescanIntervalS` to the default value when toggling the watcher is way too simplistic. It has the issue that you reported on startup (watcher is called once on startup regardless), but also if you edit folders with differing watcher setting. I'll try to solve it differently.
